app/web/features/notifications/PushNotificationBanner.tsx

```diff
@@ -0,0 +1,109 @@
+import { Alert, Button, styled } from "@mui/material";
+import { useAuthContext } from "features/auth/AuthProvider";
+import { useTranslation } from "i18n";
+import { NOTIFICATIONS } from "i18n/namespaces";
+import { usePersistedState } from "platform/usePersistedState";
+import React, { useEffect, useState } from "react";
+import { Trans } from "react-i18next";
+import { theme } from "theme";
+
+import { checkPushEnabled } from "./notificationUtils";
+import { onPushNotificationPermissionGranted } from "./utils/helpers";
+
+const TIME_BETWEEN_NAGS_MS = 180 * 86400; // 180 days
```

Comment:
this is now in seconds

Reply:
Do you wanna update the name then to what you think makes sense? Or is there something you wanted me to change?

Reply:
Yeah I'll update. Was just a note to self